MMA.md

```diff
@@ -0,0 +1,250 @@
+4. Messages:  User Interaction points that we want to engage with users that help them use Firefox better.
+
+An Event or a series of Events plus some User Attributes may trigger a Message, and when the user acts on the Message, a Deep Link may be processed.
+
+
+Data collection
+
+
+Who will have Leanplum enabled?
+======================================================
+
+Users who have a device locale listed in the following code snippet will have Leanplum enabled: https://github.com/mozilla-mobile/firefox-ios/blob/master/Client/Application/LeanplumIntegration.swift#L45
+
+
+Where does data sent to the Leanplum backend go?
+==============================================
+
+The Leanplum SDK is hard-coded to send data to the endpoint https://www.leanplum.com.  The endpoint is
+defined by ``com.leanplum.internal.Constants.API_HOST_NAME`` at
+https://searchfox.org/mozilla-central/rev/c49a70b53f67dd5550eec8a08793805f2aca8d42/mobile/android/thirdparty/com/leanplum/internal/Constants.java#32.
+
+The user is identified by Leanplum using a random UUID generated by Apple when Leanplum is initialized for the first time (see https://developer.apple.com/documentation/uikit/uidevice/1620059-identifierforvendor).
+This unique identifier is only used by Leanplum and can't be tracked back to any Firefox users.
+
+
+What data is collected and sent to the Leanplum backend?
+======================================================
+
+The Leanplum SDK collects and sends two messages to the Leanplum backend.  The messages have the
+following parameters:
+
+~~~~~~~~~~~~~~~
+
+  // Sent every time when an event is triggered
+  "action" -> "track"                   // track: an event is tracked.
+  "event" -> "Launch"                   // Used when an event is triggered. e.g. E_Saved_Bookmark.
+  "info" -> ""                          // Used when an event is triggered. Basic context associated with the event.
+  "value" -> 0.0                        // Used when an event is triggered. Value of that event.
+  "messageId" -> 5111602214338560       // Used when an event is triggered. The ID of the message.
+
+  // Sent when the app starts
+  "action" -> "start"                   // start: Leanplum SDK starts. heartbeat
+  "userAttributes" -> "{                // A set of key-value pairs used to describe the user.
+    "Focus Installed" -> true           // If Focus for iOS is installed.
+    "Klar Installed" -> true            // If Klar for iOS is installed.
+    "Signed In Sync" -> true            // If the user signed into a Firefox Account.
+    "Mailto Is Default" -> true         // If the user has not changed their default mailto handler.
+    "Telemetry Opt In" -> true          // If the user has not disabled Sending Anonymous Product Usage.
+  }"
+  "appId" -> "app_6Ao...."              // Leanplum App ID.
+  "clientKey" -> "dev_srwDUNZR...."     // Leanplum client access key.
+  "systemName" -> "Android OS"          // Fixed String in SDK.
```

Comment:
Nit: this should probably be iOS or something, not Android. Please make sure it's an actual string, not just "iOS is the platform name"